google-datacatalog-hive-connector/src/google/datacatalog_connectors/hive/scrape/metadata_database_scraper.py

```diff
@@ -32,7 +34,8 @@ def __init__(self, hive_metastore_db_host, hive_metastore_db_user,
         self.__engine = create_engine('{}://{}:{}@{}/{}'.format(
             hive_metastore_db_type, hive_metastore_db_user,
             hive_metastore_db_pass, hive_metastore_db_host,
-            hive_metastore_db_name))
+            hive_metastore_db_name),
+                                      pool_size=self.CONNECTION_POOL_SIZE)
```

Comment:
This formatting is odd... Could it be improved?

Reply:
this was generated by yapf, I can check.

Reply:
Cool, tks!